docs/Makefile

```diff
@@ -7,6 +7,7 @@ SPHINXBUILD   = sphinx-build
 SPHINXPROJ    = cuGraph
 SOURCEDIR     = source
 BUILDDIR      = build
+PDFDIR		  = pdfs
```

Comment:
pdf has a number of issues:
- git cannot generate meaningful diffs, or merge binary files.
- growing repo and most importantly history size due to the binary nature of the format.

It's fine if we never update this pdf but we should consider porting this to rst or md if we think we'll update it.

Reply:
yep.  But markdown is terrible on tables, images, formatting, etc..
I'll be replacing it in 0.17